src/controllers/expenses.controller.js

```diff
@@ -0,0 +1,133 @@
+'use strict';
+
+const { ExpenseServises } = require('../services/expenses.service');
+const { UserServices } = require('../services/users.service');
+
+const getExpenses = (req, res) => {
+  const { userId, from, to } = req.query;
+
+  const url = new URL(req.url, 'http://localhost:3000/' + req.url);
+  const categories = url.searchParams.getAll('categories');
+
+  let expenses = ExpenseServises.getExpenses();
+
+  if (userId) {
+    expenses = expenses.filter(exp => exp.userId === +userId);
```

Comment:
```suggestion
    expenses = expenses.filter(expense => expense.userId === Number(userId));
```

The same in other places